### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by bertsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +64,11 @@ Moreover, for testing we need example data (e.g. users, authorities, workflows e
 
 The fastest way to get all that is by using the Makefile via the following command:
 
-    make prepare
+    make prepare # generate all missing files
```
Comment (markusweigelt) [inline README.md:67]:
```suggestion
    make prepare # generate files and directories which are required to use our services
```

File: README.md [hunk 2]
```diff
@@ -63,7 +64,11 @@ Moreover, for testing we need example data (e.g. users, authorities, workflows e
 
 The fastest way to get all that is by using the Makefile via the following command:
 
-    make prepare
+    make prepare # generate all missing files
+    make prepare-keys # generate only SSH credentials
+    make prepare-data # generate only Kitodo OCR scripts
```
Comment (markusweigelt) [inline README.md:69]:
```suggestion
    make prepare-data # generate data which are required for all profiles

```

File: README.md [hunk 3]
```diff
@@ -63,7 +64,11 @@ Moreover, for testing we need example data (e.g. users, authorities, workflows e
 
 The fastest way to get all that is by using the Makefile via the following command:
 
-    make prepare
+    make prepare # generate all missing files
+    make prepare-keys # generate only SSH credentials
+    make prepare-data # generate only Kitodo OCR scripts
+    make prepare-examples # generate only Kitodo DB entries
```
Comment (markusweigelt) [inline README.md:70]:
```suggestion
    make prepare-examples # generate example data for Kitodo.Production for example DB entries and process
```

File: README.md (hunk 1 above)
Comment (bertsky) [reply README.md:67]: IMO shorter is better in this context. It's just the comment to the command, after all. In addition to the full explanation above that, this should just show how these are subgrouped.

File: README.md (hunk 2 above)
Comment (bertsky) [reply README.md:69]: why all profiles? It's only Kitodo, isn't it? Also, it's just the scripts I thought...

Comment (markusweigelt) [reply README.md]: Yes. Atm there are just the scripts but it is conceivable that further Kitodo files and folders will be added or service folders.

**Pull request merged**